### PR TITLE
Trim and update error messages

### PR DESCRIPTION
**Pull request by tejohnso**
Trimmed and updated error messages.  @fjvallarino and @ByronDarlison please review.

### CODE REVIEW

File: src/locales/en/storage-client.po [hunk 1]
```diff
@@ -1,47 +1,32 @@
 msgid "access-denied"
 msgstr "You don't have access to that action. Please contact your Administrator to give you the access you need."
 
-msgid "bandwidth-query-failed"
-msgstr "Bucket bandwidth query failed"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: When would a User see this, how is this relevant to them, and what should they do if it comes up?

File: src/locales/en/storage-client.po [hunk 2]
```diff
@@ -1,47 +1,32 @@
...
-
 msgid "confirm-delete"
 msgstr "Delete"
 
 msgid "confirm-deletion"
 msgstr "Please confirm PERMANENT deletion of"
 
-msgid "cost-disclaimer"
-msgstr "There is currently no charge for the use of Storage, however at an as of yet to be determined date in the future, there will be a cost and at that time you will have 90 days to decide if you wish to continue using Storage or not."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: We should no longer have a situation where this error should come up right?

File: src/locales/en/storage-client.po [hunk 3]
```diff
@@ -1,47 +1,32 @@
...
-
-msgid "create-initial-folder"
-msgstr "Create initial storage folder"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: We should no longer have a situation where this error should come up right?

File: src/locales/en/storage-client.po [hunk 4]
```diff
@@ -1,47 +1,32 @@
...
-
-msgid "create-initial-folder-failed"
-msgstr "Could not create initial folder for {{email}}"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: We should no longer have a situation where this error should come up right?

File: src/locales/en/storage-client.po [hunk 5]
```diff
@@ -58,33 +43,21 @@ msgstr "Restore Failed"
 msgid "failed-trash"
 msgstr "Move To Trash Failed"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: What should the User do if they get this message?

File: src/locales/en/storage-client.po [hunk 6]
```diff
@@ -58,33 +43,21 @@ msgstr "Restore Failed"
...
 
-msgid "folder-bucket-creation"
-msgstr "Error creating initial storage for {{email}}"
-
 msgid "folder-inactive-subscription"
-msgstr "Could not create folder. Subscription not active for {{email}}."
+msgstr "The folder could not be created because a valid subscription is required.  Please purchase a subscription and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: We need to standardize our case, in places we use sentence case, others Title Case. Trash is capitalized, folder isn't, subscription isn't, etc. For consistency everything should be Title Case;

File
Folder
Trash
Subscription
Delete
...

File: src/locales/en/storage-client.po [hunk 7]
```diff
@@ -58,33 +43,21 @@ msgstr "Restore Failed"
...
 
 msgid "folder-verify-company"
-msgstr "Could not create folder. Unable to authenticate using the email account {{email}}."
+msgstr "The folder was not created.  It appears as though account {{email}} does not have access to this company."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: Title Case

Is email = username and if it is, why don't we call this username to avoid confusion with the User's email address?

What should the User do if they get this message?

File: src/locales/en/storage-client.po [hunk 8]
```diff
@@ -92,13 +65,13 @@ msgid "new-folder"
 msgstr "New Folder"
 
 msgid "not-authorized"
-msgstr "You are not authorized to view this Company's files"
+msgstr "You are not authorized to view this Company's files.  Please try logging in again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: Shouldn't this be consistent with the above where we state which User account we see them using? And that it isn't authorized to view this Company's files? Or is this something different? And if not, shouldn't they try logging in with the username that belongs to this Company?

File: src/locales/en/storage-client.po (hunk 5 above)
Comment (tejohnso) [inline src/locales/en/storage-client.po]: This is a case where there is some unexpected error during the process.  Will update the message.

File: src/locales/en/storage-client.po [hunk 9]
```diff
@@ -92,13 +65,13 @@ msgid "new-folder"
...
 
 msgid "pending-delete"
 msgstr "Deleting"
 
 msgid "pending-ops-leave-page"
-msgstr "Warning: There are operations pending."
+msgstr "Warning: There are operations pending.  If you close this window some tasks will remain incomplete."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: What is an operation? How does a User know what that means? And curious, can't this work be done server side? Why does it have to rely on the client?

File: src/locales/en/storage-client.po [hunk 10]
```diff
@@ -119,7 +92,7 @@ msgid "requesting-permission"
 msgstr "Requesting permission"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: When would this message appear?

File: src/locales/en/storage-client.po [hunk 11]
```diff
@@ -119,7 +92,7 @@ msgid "requesting-permission"
...
 
 msgid "restore-inactive-subscription"
-msgstr "Could not restore from trash. Subscription not active for {{email}}"
+msgstr "The restore request was refused because a valid subscription is required.  Please purchase a subscription and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: You cannot recover the File from Trash because you don't have an Active Subscription. Please purchase a Subscription and try again.

File: src/locales/en/storage-client.po (hunk 6 above)
Comment (tejohnso) [inline src/locales/en/storage-client.po]: Are you saying that the word subscription should be capitalized in the above two sentences?  And that those 5 words should always be capitalized?

File: src/locales/en/storage-client.po [hunk 12]
```diff
@@ -127,17 +100,11 @@ msgstr "Restored From Trash"
...
 
-msgid "server-error"
-msgstr "There was a problem with the request. Report to support with this error code: "
-
 msgid "signed-download-uri-request-failed"
-msgstr "Download request failed for {{email}}"
-
-msgid "signed-url-inactive-subscription"
-msgstr "Could not Download. Subscription not active for {{email}}."
+msgstr "There was an unexpected server error during the download request.  Please try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: The User doesn't care if it was a server error, the downloaded was requested, etc.

There was a download error. Please try again.

File: src/locales/en/storage-client.po [hunk 13]
```diff
@@ -127,17 +100,11 @@ msgstr "Restored From Trash"
...
 
 msgid "signed-url-verify-company"
-msgstr "Could not Download. We're unable to authenticate you using the email account {{email}}."
+msgstr "The download request was refused because email account {{email}} does not have access to this company.  Please log in and retry."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: The download failed because your account {{email}} doesn't have access to this Company. Please login with a different account and try again.

File: src/locales/en/storage-client.po [hunk 14]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
 
 msgid "trash-inactive-subscription"
-msgstr "Could not move to trash. Subscription not active for {{email}}"
-
-msgid "unexpected-null-parameter"
-msgstr "Unexpected null parameter"
-
-msgid "upload-bucket-creation"
-msgstr "Error creating initial storage for {{email}}"
+msgstr "The trash operation was refused because a valid subscription is required.  Please purchase a subscription and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: You cannot move this File to Trash because you don't have an Active Subscription. Please purchase a Subscription and try again.

File: src/locales/en/storage-client.po [hunk 15]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
 
 msgid "upload-failed"
-msgstr "Upload did not complete"
+msgstr "There was an unexpected error during upload.  Please try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: There was a upload error. Please try again.

File: src/locales/en/storage-client.po [hunk 16]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
 
 msgid "upload-file"
 msgstr "Upload File"
 
 msgid "upload-inactive-subscription"
-msgstr "Could not Upload. Subscription not active for {{email}}."
+msgstr "The upload request was rejected because a valid subscription is required.  Please purchase a subscription and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: You cannot Upload Files because you don't have an Active Subscription. Please purchase a Subscription and try again.

File: src/locales/en/storage-client.po (hunk 7 above)
Comment (tejohnso) [inline src/locales/en/storage-client.po]: @fjvallarino Can you go through all of the .po, html and js, files and change occurrences of {{email}} to {{username}}?  I'm not sure of all the different places it shows up.  It looks like it's only in one service in storage-client but that doesn't seem right.

File: src/locales/en/storage-client.po [hunk 17]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
 
 msgid "upload-status"
 msgstr "Upload Status"
 
 msgid "upload-uri-request-failed"
-msgstr "Upload URI request failed"
-
-msgid "upload-uri-request-failed-mail"
-msgstr "Upload URI request failed for {{email}}"
+msgstr "There was an unexpected problem processing the upload.  Please try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: There was a Upload error. Please try again.

File: src/locales/en/storage-client.po [hunk 18]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
 
 msgid "upload-verify-company"
-msgstr "Could not upload.  Unable to authenticate using the email account {{email}}."
+msgstr "The upload request was refused because account {{email}} does not have access to this company.  Please log in and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: The Upload failed because your Account {{email}} doesn't have access to this Company. Please login with a different Account and try again.

File: src/locales/en/storage-client.po (hunk 9 above)
Comment (tejohnso) [inline src/locales/en/storage-client.po]: An operation could be anything like pending uploads, or pending deletes, or pending trash moves.
If the user has 200 operations pending, the client knows about it because it submits them one after the other as they complete.  The server doesn't know more ops are coming.  It should be obvious to the user what an operation is given the context of their UI use.  They'll know what they've got pending because the UI shows them.  The message is just to prevent them from closing the window prematurely.

File: src/locales/en/storage-client.po (hunk 7 above)
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: You cannot create a Folder because your Account {{email}} doesn't have access to this Company. Please login with a different Account and try again.

File: src/locales/en/storage-client.po (hunk 10 above)
Comment (tejohnso) [inline src/locales/en/storage-client.po]: Just after you select a file to upload.  It briefly says "requesting permission" before it changes to "uploading ....filename"

File: src/locales/en/storage-client.po (hunk 8 above)
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: Your Account {{email}} doesn't have access to this Company. Please login with a different Account and try again.

File: src/locales/en/storage-client.po (hunk 9 above)
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: There are tasks in progress. If you close this window they won't be completed.

File: src/locales/en/storage-client.po (hunk 7 above)
Comment (fjvallarino) [inline src/locales/en/storage-client.po]: @tejohnso ctr-buttons.js:287 also uses email

File: src/locales/en/storage-client.po (hunk 10 above)
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: Why does the User need to see that? All they care about is that they are uploading a file. What we have to do to make that happen isn't relevant to them. Keep it simple. They should see "Uploading ... filename."  And that's it.

File: src/locales/en/storage-client.po [hunk 19]
```diff
@@ -1,47 +1,32 @@
...
-
-msgid "create-initial-folder-failed"
-msgstr "Could not create initial folder for {{email}}"
-
 msgid "delete-file"
 msgstr "file: {{file}}"
 
 msgid "delete-files-folders-plural"
-msgstr "{{count}} files and folders are about to be permanently deleted and they cannot be recovered."
+msgstr "{{count}} files and folders are about to be permanently deleted."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: "{{count}} Files and Folders are about to be permanently deleted."

File: src/locales/en/storage-client.po [hunk 20]
```diff
@@ -1,47 +1,32 @@
...
 
 msgid "delete-files-plural"
-msgstr "{{count}} files are about to be permanently deleted and they cannot be recovered."
+msgstr "{{count}} files are about to be permanently deleted."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: "{{count}} Files are about to be permanently deleted."

File: src/locales/en/storage-client.po [hunk 21]
```diff
@@ -1,47 +1,32 @@
...
 
 msgid "delete-files-singular"
-msgstr "One file is about to be permanently deleted and it cannot be recovered."
+msgstr "One file is about to be permanently deleted."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: "One File is about to be permanently deleted."

File: src/locales/en/storage-client.po [hunk 22]
```diff
@@ -1,47 +1,32 @@
...
 
 msgid "delete-folder"
 msgstr "folder: {{folder}} and all its contents"
 
 msgid "delete-folders-plural"
-msgstr "{{count}} folders are about to be permanently deleted and they cannot be recovered."
+msgstr "{{count}} folders are about to be permanently deleted."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: "{{count}} Folders are about to be permanently deleted."

File: src/locales/en/storage-client.po [hunk 23]
```diff
@@ -1,47 +1,32 @@
...
 
 msgid "delete-folders-singular"
-msgstr "One folder is about to be permanently deleted and it cannot be recovered."
-
-msgid "delete-inactive-subscription"
-msgstr "Could not delete. Subscription not active for {{email}}"
+msgstr "One folder is about to be permanently deleted."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: "One Folder is about to be permanently deleted."

File: src/locales/en/storage-client.po [hunk 24]
```diff
@@ -56,35 +41,23 @@ msgid "failed-restore"
 msgstr "Restore Failed"
 
 msgid "failed-trash"
-msgstr "Move To Trash Failed"
-
-msgid "folder-bucket-creation"
-msgstr "Error creating initial storage for {{email}}"
+msgstr "There was an unexpected error during the trash operation.  Please try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: Moving to Trash failed. Please try again.

File: src/locales/en/storage-client.po [hunk 25]
```diff
@@ -56,35 +41,23 @@ msgid "failed-restore"
...
 
 msgid "folder-inactive-subscription"
-msgstr "Could not create folder. Subscription not active for {{email}}."
+msgstr "The folder could not be created because a valid subscription is required.  Please purchase a subscription and try again."
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: You cannot add a Folder because you don't have an Active Subscription. Please purchase a Subscription and try again.

File: src/locales/en/storage-client.po [hunk 26]
```diff
@@ -149,41 +116,28 @@ msgid "trash"
...
+msgstr "The upload request was refused because account {{email}} does not have access to this company.  Please log in and try again."
 
 msgid "uploading"
 msgstr "Uploading {{filename}}"
 
-msgid "verify-failed"
-msgstr "Could not verify"
-
 msgid "verifying"
 msgstr "Verifying {{filename}}"
```
Comment (ByronDarlison) [inline src/locales/en/storage-client.po]: When do we need to verify something?
